test-cases/gemini/gemini-3h-cdc-write.yaml

```diff
@@ -0,0 +1,25 @@
+test_duration: 300
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+instance_type_db: 'i3.large'
+
+store_results_in_elasticsearch: False
+user_prefix: "gemini-cdc-write"
+
+gemini_cmd: "gemini -d --duration 3h \
+-c 50 -m write -f --non-interactive \
+--cql-features all \
+--replication-strategy \"{'class': 'SimpleStrategy', 'replication_factor': '3'}\" "
+
+gemini_table_options:
+  - "cdc={'enabled': true}"
```

Comment:
Can we enable preimage, postimage? are they supported by C*?

Reply:
Yes it should be enabled in gemini_table_options config parameter. Missed it in preimage.
Fixed